components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3233,6 +3233,9 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     async getStripePortalUrlForTeam(ctx: TraceContext, teamId: string): Promise<string> {
         throw new ResponseError(ErrorCodes.SAAS_FEATURE, `Not implemented in this version`);
     }
+    async getStripePortalUrl(ctx: TraceContext, attributionId: string): Promise<string> {
```

Comment:
Also, tiny problem, the order is wrong 😇

```suggestion
    async getStripePortalUrlForUser(ctx: TraceContext, attributionId: string): Promise<string> {
```

Reply:
> Also, tiny problem, the order is wrong

I'm not sure what you mean by this 😕

Reply:
Sorry for the noise -- I was mistaken, which is why I immediately resolved this conversation. 🙈